node-api/src/error.rs

```diff
@@ -23,92 +23,55 @@ use crate::{
     events::EventsDecodingError,
     metadata::{InvalidMetadataError, Metadata, MetadataError},
 };
+use derive_more::From;
 use sp_core::crypto::SecretStringError;
 use sp_runtime::{transaction_validity::TransactionValidityError, DispatchError, ModuleError};
-use thiserror::Error;
+
+#[cfg(not(feature = "std"))]
+use alloc::string::{String, ToString};
+
+#[cfg(not(feature = "std"))]
+use sp_std::vec::Vec;
```

Comment:
in `node-api/src/lib.rs` you're using the alloc vec. Would probably make sense to stay consistent.